glib/src/value.rs

```diff
@@ -1224,6 +1224,15 @@ mod tests {
         assert_eq!(v2.get::<&str>(), Ok(Some("123")));
     }
 
+    #[test]
+    fn test_into_raw() {
+        unsafe {
+            let v = 123.to_value().into_raw();
+            assert_eq!(gobject_ffi::g_type_check_value(&v), ffi::GTRUE);
+            assert_eq!(gobject_ffi::g_value_get_int(&v), 123);
```

Comment:
Should probably for completeness also call `gobject_ffi::g_value_unset(&mut v)` here